.github/workflows/build_preview.yaml

```diff
@@ -61,32 +65,44 @@ jobs:
           echo $ver
           echo "hugovr=$ver" >> $GITHUB_ENV
 
-      - name: Setup Hugo
-        uses: peaceiris/actions-hugo@v2
-        with:
-          hugo-version: ${{ env.hugovr }}
-          extended: true
-      
-      - name: Setup R
-        uses: r-lib/actions/setup-r@v2
+      - name: Install cURL Headers
+        run: sudo apt-get install libcurl4-openssl-dev
         
-      - name: Install R-packages
-        run: |
-          install.packages(c(
-            "here", "jsonlite",
-            "dplyr", "tidyr"),
-            Ncpus = 2)
-        shell: Rscript {0}
+      - uses: r-lib/actions/setup-r@v2
+      - uses: r-lib/actions/setup-renv@v2    
 
       - name: Populate untranslated pages
         run: Rscript scripts/missing_translations.R
         
-      - name: Site data - Get directory data
+      - name: Site data - Clean folders
+        run: rm -r data/directory data/rblogs
+        
+      - name: Site data - DIRECTORY - Get
         uses: actions/checkout@v3
+        if: not ${{ github.event.inputs.directory_artifact }}
         with:
           repository: rladies/directory
           ssh-key: ${{ secrets.DIRECTORY_DEPLOY_SECRET}}
           path: tmpd/dir
+
+      - name: Site data  - DIRECTORY - Download artifact
+        if: ${{ github.event.inputs.directory_artifact }}
```

Comment:
not I'm wishing there were an if else thing in GHA!

Reply:
yes, that would have been nice to have within a step!